unikname_connect.php

```diff
@@ -1264,9 +1266,9 @@ function the_champ_create_admin_menu(){
 	// add_action('admin_print_scripts-' . $facebookPage, 'the_champ_admin_scripts');
 	// add_action('admin_print_scripts-' . $facebookPage, 'the_champ_fb_sdk_script');
 	// add_action('admin_print_styles-' . $facebookPage, 'the_champ_admin_style');
-	add_action('admin_print_scripts-' . $loginPage, 'the_champ_admin_scripts');
-	add_action('admin_print_scripts-' . $loginPage, 'the_champ_fb_sdk_script');
-	add_action('admin_print_styles-' . $loginPage, 'the_champ_admin_style');
+	// add_action('admin_print_scripts-' . $loginPage, 'the_champ_admin_scripts');
+	// add_action('admin_print_scripts-' . $loginPage, 'the_champ_fb_sdk_script');
+	// add_action('admin_print_styles-' . $loginPage, 'the_champ_admin_style');
```

Comment:
What is this?

Reply:
We have fully customized the admin so we don't need it. If there is an error "$loginPage" undefined